src/main/java/org/cactoos/http/HtCookies.java

```diff
@@ -52,8 +54,8 @@ public HtCookies(final Input rsp) {
         super(() -> {
             final Map<String, String> result = new HashMap<>();
             final String cookie = new HtHeaders(rsp).get("set-cookie");
-            for (final String item : cookie.split(";\\s+")) {
-                final String[] entry = item.split("=", 2);
+            for (final Text item : new SplitText(cookie, ";\\s+")) {
```

Comment:
@VsSekorin Sorry, it seems I was no so clear :) This code is still too procedural and it can be easily refactored: just try to get rid of the loop using Mapped. It should look like that:
```!java
final Map<String, String> result = new MapOf<>(
    new Mapped<>(
        input -> {
            final Iterator<Text> parts =
                new SplitText(input, "=").iterator();
            return new MapEntry<>(
                parts.next().asString(),
                parts.next().asString()
            );
        },
        new IterableOf<>(
            new SplitText(cookie, ";\\s+").iterator()
        )
    )
);
```
Please fix it (you need to handle an error case about "Incorrect HTTP Response cookie").